plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_config.py

```diff
@@ -58,7 +58,9 @@ class RayProviderConf:
     region: str = "us-west-2"
     availability_zone: str = "us-west-2a,us-west-2b"
     cache_stopped_nodes: bool = False
-    key_pair: Dict[str, str] = field(default_factory=lambda: {"key_name": "hydra"})
+    key_pair: Dict[str, str] = field(
+        default_factory=lambda: {"key_name": "hydra-${env:USER,me}"}
```

Comment:
Why is "me" a good default value for a key name?

Reply:
probably not a good name :) how about just `user`?

Reply:
why not hydra?

Reply:
right now the key name is hydra-{$USER}, are you saying is $USER does not exist we do hydra-hydra?

Reply:
I'm going to name it `hydra-${env:USER, user}` then :)